crates/router/src/core/utils.rs

```diff
@@ -543,6 +577,43 @@ pub async fn construct_retrieve_file_router_data<'a>(
         reference_id: None,
         payment_method_token: None,
         preprocessing_id: None,
+        connector_request_reference_id: "irrelevant_connector_request_reference_id_in_dispute_flow"
+            .to_string(),
     };
     Ok(router_data)
 }
+
+pub async fn get_connector_request_reference_id(
+    db: &dyn db::StorageInterface,
+    merchant_id: &str,
+    payment_attempt: &storage_models::payment_attempt::PaymentAttempt,
+) -> String {
+    let lookup_key = SEND_PAYMENT_ID_AS_CONNECTOR_REQUEST_REFERENCE_ID;
+    let option_config_map = match db.find_config_by_key(lookup_key).await {
```

Comment:
You can use this 
https://github.com/juspay/hyperswitch/blob/e913bfc4958da613cd352eca9bc38b23ab7ac38e/crates/router/src/db/configs.rs#L88